rhwp/rhwp-studio/src/ui/agent-sidebar/index.ts

```
@@ -4390,6 +4351,9 @@ export function initAgentSidebar(deps: AgentSidebarDeps): {
       case 'chat-started':
         if (e.threadId && e.threadId !== currentThread.id) break;
         chatStartPendingThreadId = null;
+        const prevAgent = selectedAgent;
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)